godel_process_execution/src/blend_process_service_node.cpp

```diff
@@ -0,0 +1,35 @@
+#include <godel_process_execution/blend_process_service.h>
+
+#include <ros/ros.h>
+
+const static std::string DEFAULT_SERVICE_NAME = "blend_process_execution";
+
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "blend_process_service_node");
+
+  ros::NodeHandle nh;
+  ros::NodeHandle pnh ("~");
+
+  std::string sim_service, execution_service;
+
+  if (!pnh.getParam("execution_service", execution_service))
+  {
+    ROS_ERROR("Blend Process Execution node requires the 'execution_service' parameter be set.");
+    return -1;
+  }
+
+  if (!pnh.getParam("simulation_service", sim_service))
```

Comment:
I think that passing names of topics and services through parameters is overkill as you could easily change these through `remap` in the corresponding launch file.